code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -152,6 +156,11 @@
 		use_power = 0
 	else if(!stat && construction_state <= 3)
 		use_power = 1
+	src.update_icon()
+	for(var/obj/structure/particle_accelerator/part in connected_parts)
+		part.strength = null
+		part.powered = 0
+		part.update_icon()

```

Comment:
I'm pretty sure `power_change` shouldn't do anything if the state of power didn't change